chain-tx-enclave-next/mls/src/tree.rs

```diff
@@ -570,6 +570,28 @@ impl TreePublicKey {
         }
         self.nodes[path[0].node_index()].compute_parent_hash(self.cs)
     }
+
+    fn verify_node_private_key(
```

Comment:
is it ok to make this pub(crate)? I may need to use this -- just document the intended usage/args

Reply:
done